dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -91,27 +92,70 @@ const stickyContainerStyles = (height: number) => css`
 interface Props {
 	isPlaying: boolean;
 	children: React.ReactNode;
+	id: string;
 }
 
-export const StickyVideo = ({ isPlaying, children }: Props) => {
-	const [isSticky, setIsSticky] = useState(false);
+interface StickyState {
+	isSticky: boolean;
+	isClosed: boolean;
+}
+
+export const StickyVideo = ({ isPlaying, children, id }: Props) => {
+	const [stickyState, setStickyState] = useState<StickyState>({
+		isSticky: false,
+		isClosed: false,
+	});
+
 	const [isIntersecting, setRef] = useIsInView({
 		threshold: 0.5,
 		repeat: true,
 	});
 
 	useEffect(() => {
-		setIsSticky(isPlaying && !isIntersecting);
+		setStickyState({
+			isSticky: isPlaying && !isIntersecting,
+			isClosed: false,
+		});
 	}, [isIntersecting, isPlaying]);
 
+	useEffect(() => {
+		const ophanTracking = (trackingEvent: OphanAction) => {
```

Comment:
If we remove the early return for `id` we don't need this function at all, I guess? So then we can just inline the tracking requests (as I'm sure you're aware of by now, I'm a fan of inlining code!)

Reply:
Good suggestion. By removing the id check, now I'm calling `submitComponentEvent` in line wherever it needs to emit tracking events.